### ISSUE TITLE
vetur server fails to compile

### ISSUE THREAD

**Issue by ntraut** [vuejs/vetur#2860]
- [x] I have searched through existing issues
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://vuejs.github.io/vetur/guide/FAQ.html)
- [x] I have tried restarting VS Code or running `Vetur: Restart VLS`

## Info

- Platform: Win 10
- Vetur version: git commit 317a44fe48edbf
- VS Code version: 1.55.1

## Problem

When I try to compile Vetur following [the wiki instructions](https://github.com/vuejs/vetur/wiki/Developing-Language-Server-Features) (`yarn && yarn compile`) I get these errors:
```
D:\vetur\server\src\vueServerMain.ts → server\dist\vueServerMain.js...
 > server/src/services/vls.ts:162:8: warning: This call to "require" will not be bundled because the argument is not a string literal (surround with a try/catch to silence this warning)
    162 │         require(path.join(workspace.fsPath, '.pnp.js')).setup();
        ╵         ~~~~~~~

 > server/src/services/vls.ts:164:8: warning: This call to "require" will not be bundled because the argument is not a string literal (surround with a try/catch to silence this warning)
    164 │         require(path.join(workspace.fsPath, '.pnp.cjs')).setup();
        ╵         ~~~~~~~

 > server/src/utils/workspace.ts:9:9: warning: This call to "require" will not be bundled because the argument is not a string literal (surround with a try/catch to silence this warning)
    9 │   return require(module);
      ╵          ~~~~~~~

 > server/src/services/dependencyService.ts:127:21: warning: This call to "require" will not be bundled 
because the argument is not a string literal (surround with a try/catch to silence this warning)        
    127 │     const tsModule = require(dir);
        ╵                      ~~~~~~~

 > server/src/services/dependencyService.ts:243:16: warning: This call to "require" will not be bundled 
because the argument is not a string literal (surround with a try/catch to silence this warning)        
    243 │         module: require(pkgPath)
        ╵                 ~~~~~~~

 > server/node_modules/vue-eslint-parser/index.js:659:12: warning: This call to "require" will not be bundled because the argument is not a string literal (surround with a try/catch to silence this warning)  
    659 │             require(parserOptions.parser)
        ╵             ~~~~~~~

 > server/node_modules/prettier-eslint/dist/index.js:113:89: warning: "@typescript-eslint/parser" should be marked as external for use with "require.resolve"
    113 │ ....parser = formattingOptions.eslint.parser || require.resolve('@typescript-eslint/parser'); 
        ╵                                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~   

 > server/node_modules/prettier-eslint/dist/index.js:117:89: warning: "vue-eslint-parser" should be marked as external for use with "require.resolve"
    117 │ ...s.eslint.parser = formattingOptions.eslint.parser || require.resolve('vue-eslint-parser');
        ╵                                                                         ~~~~~~~~~~~~~~~~~~~

 > server/src/modes/vue/snippets.ts:155:4: warning: This case clause will never be evaluated because it 
duplicates an earlier case clause
    155 │     case 'template':
        ╵     ~~~~

 > server/node_modules/stylus/lib/utils.js:132:15: warning: This call to "require" will not be bundled because the argument is not a string literal (surround with a try/catch to silence this warning)
    132 │     var main = require(relative(__dirname, pkg)).main;
        ╵                ~~~~~~~

 > server/node_modules/stylus/lib/functions/selector.js:60:19: warning: Indirect calls to "require" will not be bundled (surround with a try/catch to silence this warning)
    60 │   var Parser = new require('../parser')
       ╵                    ~~~~~~~

 > server/node_modules/stylus/lib/functions/use.js:29:11: warning: This call to "require" will not be bundled because the argument is not a string literal (surround with a try/catch to silence this warning)  
    29 │   var fn = require(path.resolve(found));
       ╵            ~~~~~~~

 > server/node_modules/prettier-tslint/node_modules/yargs/index.js:12:39: warning: Indirect calls to "require" will not be bundled (surround with a try/catch to silence this warning)
    12 │   const argv = yargs(processArgs, cwd, require)
       ╵                                        ~~~~~~~

 > server/node_modules/prettier-tslint/node_modules/yargs/yargs.js:350:33: warning: Indirect calls to "require" will not be bundled (surround with a try/catch to silence this warning)
    350 │     const req = parentRequire || require
        ╵                                  ~~~~~~~

 > server/node_modules/prettier-tslint/node_modules/yargs/lib/apply-extends.js:43:82: warning: This call to "require" will not be bundled because the argument is not a string literal (surround with a try/catch to silence this warning)
    43 │ ...fig = isPath ? JSON.parse(fs.readFileSync(pathToDefault, 'utf8')) : require(config.extends)
       ╵                                                                        ~~~~~~~

 > server/node_modules/require-main-filename/index.js:2:25: warning: Indirect calls to "require" will not be bundled (surround with a try/catch to silence this warning)
    2 │   _require = _require || require
      ╵                          ~~~~~~~

16 warnings
✨ success with esbuild
node_modules/@types/eslint/index.d.ts:299:41 - error TS2724: '"D:/github/vetur/server/node_modules/@types/estree/index"' has no exported member named 'ChainExpression'. Did you mean 'ThisExpression'? 

299         ChainExpression?: (node: ESTree.ChainExpression & NodeParentExtension) => void;
                                            ~~~~~~~~~~~~~~~

node_modules/@types/eslint/index.d.ts:322:42 - error TS2694: Namespace '"D:/github/vetur/server/node_modules/@types/estree/index"' has no exported member 'ImportExpression'.

322         ImportExpression?: (node: ESTree.ImportExpression & NodeParentExtension) => void;
                                             ~~~~~~~~~~~~~~~~


Found 2 errors.

[!] (plugin generate-typings-vls) Error: type-check error.
Error: type-check error.
    at error (D:\github\vetur\node_modules\rollup\dist\shared\rollup.js:5305:30)
    at throwPluginError (D:\github\vetur\node_modules\rollup\dist\shared\rollup.js:18033:12)    
    at D:\github\vetur\node_modules\rollup\dist\shared\rollup.js:18865:27
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Promise.all (index 1)
    at rollupInternal (D:\github\vetur\node_modules\rollup\dist\shared\rollup.js:20206:9)       
    at build (D:\github\vetur\node_modules\rollup\dist\bin\rollup:1480:20)
    at runRollup (D:\github\vetur\node_modules\rollup\dist\bin\rollup:1662:21)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
## Reproducible Case

Just do `git clone https://github.com/vuejs/vetur` and then the commands above.

**Comment by ntraut** [comment]
Looking at the results of the CI the problem appeared on the commit 53d3df0.

**Comment by softwaredeveloptam** [comment]
This is also happening to me on the recent pull from master.

**Comment by yoyo930021** [comment]
Fixed in da59c05
